ci/scripts/run-skyhook-test

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+set -e
+
+# copy test data
+cp -r data /data
+
+# launch cluster
+cd build/
+MON=1 OSD=1 ../src/vstart.sh -d -n -x
+
+# run test
+bin/ceph_test_skyhook_query
```

Comment:
Don't we also need to copy the CLS classes into the proper directory?

Reply:
same as above, binaries are in the workspace (in the `build/` folder), so they're picked up by the test